### ISSUE TITLE
Are we good to cut a patch release?

### ISSUE THREAD

**Issue by andrewculver** [bullet-train-co/nice_partials#50]
After the latest improvements from @kaspth, I can remove `<% yield p = np %>` across the whole Bullet Train codebase and the test suite passes when bundling `nice_partials` from `main`. Wanted to check with other folks like @domchristie and @seanpdoyle that you're not seeing any regressions when you bundle from `main`.

**Comment by andrewculver** [comment]
Also, probably worth asking as well: Once you've confirmed your test suite is passing, it would also be helpful if you could remove all the `<% yield p = np %>` invocations at the top of your partials and see whether the test suite still passes.

**Comment by kaspth** [comment]
You can use this updating script to remove the `<% yield p = np %>` and replace `p.x` calls with `partial.x`:

```ruby
files_to_inspect = []

Dir["app/views/**/*.html.erb"].each do |path|
  if contents = File.read(path).match(/(<%=? yield\(?.*? = np\)? %>\n+)/m)&.post_match
    files_to_inspect << path if contents.match?(/render.*?do \|/)

    contents.gsub! /\bp\.(?=yield|helpers|content_for|content_for\?)/, "partial."
    File.write path, contents
  end
end

if files_to_inspect.any?
  puts "These files had render calls with a block parameter and likely require some manual edits:"
  puts files_to_inspect
else
  puts "No files with renders calls with a block parameter found, you're likely all set"
end
```

Originally posted here https://github.com/bullet-train-co/nice_partials/issues/34#issuecomment-1173077601

**Comment by kaspth** [comment]
The patch release was shipped in v0.1.9, so we can close this out.
